openwisp_monitoring/device/tests/test_models.py

```diff
@@ -578,6 +578,90 @@ def _create_env(self):
         )
         return dm, ping, load, process_count
 
+    def test_disabling_critical_check(self):
+
```

Comment:
Remove this blank line and any other blank line in **all these tests that you added** to maintain consistency with the rest of the test of this document. Occasional blank lines are fine but if you need to add so many, you probably need to increase the font size and/or line height of your code editor.